dynamallow/model.py

```diff
@@ -141,15 +148,15 @@ def say_hello(self):
                 ))
     """
 
-    def __init__(self, raw=None, **kwargs):
+    def __init__(self, **raw):
```

Comment:
Actually, i think this is okay - we are keeping the raw data around and validating inside here - so `self._data` is actually the validated dictionary.  So technically we have access to both. We also set key/value pairs from the validated dict on the class.